### PR TITLE
Simplified friends model, added friends api.

### PR DESCRIPTION
**Pull request by ProCynic**
also removed Application.about in favor of Users.about

### CODE REVIEW

File: app/views/Events/event.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
   <h4>Going(${event.getMemberCount()}):</h4>
   #{list items:event.members, as:'member'}
-    <a href="@{Application.about(member.id)}">${member.name}</a>   
+    <a href="/users/${member.id}">${member.name}</a>
```
Comment (jtessler) [inline app/views/Events/event.html:35]: I'd use `@{Users.about(member.id)}` to generate the URL. If we want to update `conf/routes`, we shouldn't have to change any templates. This arises a few times in this pull request.

Comment (ProCynic) [inline app/views/Events/event.html:35]: I think the routes are more constant that the method names.
-Geoffrey Parker

On Fri, Apr 27, 2012 at 7:08 PM, Joe Tessler <
reply@reply.github.com

Comment (ProCynic) [inline app/views/Events/event.html:35]: I'm trying to RESTify the routes file, which means that they are, or will be, pretty much fixed.  /users/2 will always be the page about bob, whether that's Application.about(2) or Users.about(2) or the timeline, wherever that is.

Comment (jtessler) [inline app/views/Events/event.html:35]: Hmm, I'm not sure I agree. I find `@{Users.about(member.id)}` cleaner and more readable than `/users/${member.id}`. Both approaches are _eventually_ just as "constant," but I think the former is more modular. I'd like to follow the discipline that the `conf/` directory represents what we can "configure," but using `/users/${member.id}` breaks that capability.

Comment (ProCynic) [inline app/views/Events/event.html:35]: I think it's _less_ modular.  As long as we have users `/users/{userId}` will **always** be the URI of a specific user.  Using that in the links means that we can make all the changes we like to the controllers and never have to touch the links in the html.  We only have to change the controller in `conf/routes`.  Whereas if we go from `Application.about()` to `Users.about()` we have to find all the links in the html and change them.  I had to grep the html to make sure I didn't miss any, but now that it's set to `/users/{userId}` we'll never have to worry about it again.

**Comment by ProCynic** [comment]
Well, this has been up for a day and I haven't got any strong objections, so I'm merging it.

**Pull request merged**